src/components/listOfLinkTeasers/formatted-type.ts

```diff
@@ -1,8 +1,10 @@
 import { PublishedParagraph } from '@/types/formatted/publishedEntity'
-import { LinkTeaser } from '@/components/linkTeaser/formatted-type'
+import { FormattedLinkTeaser } from '@/components/linkTeaser/formatted-type'
 
 export type ListOfLinkTeasers = PublishedParagraph & {
   type: 'paragraph--list_of_link_teasers'
   title: string
-  linkTeasers: LinkTeaser[]
+  linkTeasers: FormattedLinkTeaser[]
+  parentField?: string | null
+  isHubPage: boolean | false
 }
```

Comment:
Likewise shouldn't need to change this module, but the `ListOfLinkTeasers` component function's props would have to change to add the `isHubPage` prop, which can be done just in the template file so it doesn't change the expected shape of the data.